### DISCUSSION TITLE
How to use data attribute in Oxpecker.Solid?

### DISCUSSION THREAD

**Discussion by octaviordz** [Lanayx/Oxpecker#37] [Q&A] [answered]
How to use data attribute in Oxpecker.Solid?

I tried adding the attribute to the code like:
```
li (
    class' = "drag-item",
    draggable = "true",
    dataValue = ""
) 
```
Or:
```
li (
    class' = "drag-item",
    draggable = "true"
) .data("value", "")
```
But that did not work.

**Comment by Lanayx** [comment, marked as answer]
Hi! You need to remove the space between `li` and `(`

**Comment by octaviordz** [comment]
Hi @Lanayx , 

Thank you for your response that works.

For reference:
```
li().data("value", "")
```

And for general information, I'm using Fantomas to format the F# code and Fantomas adds the space. But after removing the space and adding `.data` it stops adding the space 👍.

**Reply by Lanayx** [reply]
Great that this helped! I personally don't use Fantomas for views, since I don't like how it formats them.
